internal/service/cluster/resource_cluster_test.go

```diff
@@ -1451,9 +1451,9 @@ func TestAccClusterRSCluster_withDefaultBiConnectorAndAdvancedConfiguration_main
 				ConfigPlanChecks: resource.ConfigPlanChecks{
 					PostApplyPreRefresh: []plancheck.PlanCheck{
 						acc.DebugPlan(),
+						plancheck.ExpectEmptyPlan(),
 					},
 				},
-				PlanOnly: true,
```

Comment:
Should we a quick search and reply in the repo? 👀

Reply:
sorry I did not understand this point

Reply:
just updated the message. I am wondering if we should search for `PlanOnly` in the repo and remove it

Reply:
yeah, was waiting for confirmation [here](https://github.com/hashicorp/terraform-plugin-testing/issues/256#issuecomment-1870281163) as the migration test documentation still mentions `PlanOnly` usage. Can do this in a followup PR.